docker/tests/checkLinks.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e -x
+
+export LC_ALL=C.UTF-8
+hugo -D
+cd public
+htmlproofer --url-ignore "#","http://prometheus-istio-system.127.0.0.1.nip.io","http://getgrav.org" --assume-extension --check-external-hash .
```

Comment:
I think this should live in the maistra.github.io repo, made accessible by e.g. a Makefile. Reason being that things like the `url-ignore` strongly depend on content

Reply:
Thanks. Fixed in Maistra/maistra.github.io#123